lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -431,8 +519,8 @@ class RenderPopover extends RenderShiftedBox {
   /// Computes the (x, y) offset used to paint the menu content inside the popover.
   Offset _computeContentOffset(double arrowLength) {
     return Offset(
-      (padding?.left ?? 0) + arrowLength,
-      (padding?.top ?? 0) + arrowLength,
+      (padding?.left ?? 0) + (_reservedSizeForArrow.width / 2.0),
```

Comment:
Please add a comment about why you're dividing by `2.0` - that will surely confuse people without adding context.